tensorflow_probability/python/distributions/categorical.py

```diff
@@ -30,6 +30,7 @@
 from tensorflow_probability.python.internal import samplers
 from tensorflow_probability.python.internal import tensor_util
 from tensorflow_probability.python.internal import tensorshape_util
+from tensorflow_probability.python.math import reduce_weighted_logsumexp
```

Comment:
Please import as

```
from tensorflow_probability.python.math.generic import reduce_weighted_logsumexp
```

and add this to the "categorical" `deps` list in the adjacent BUILD file:

"//tensorflow_probability/python/math:generic"